dashboard/cron.py

```diff
@@ -52,7 +52,11 @@ def util_function(UDW_course_id, sql_string, mysql_table, table_identifier=None)
     logger.debug(" table: " + mysql_table + " insert size: " + str(df.shape[0]))
 
     # write to MySQL
-    df.to_sql(con=engine, name=mysql_table, if_exists='append', index=False)
+    try:
+        df.to_sql(con=engine, name=mysql_table, if_exists='append', index=False)
+    except Exception as e:
+        logger.warn(e)
```

Comment:
I feel this should be logged as an error and not warn.

Reply:
Yup! I changed this to logger.exception which logs as an error and prints a message.

Reply:
It prints out something like 

```
[2018-12-11 13:16:23 -0500] [34] [ERROR] Error running to_sql on table user
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1170, in _execute_context
    context)
  File "/usr/local/lib/python3.6/site-packages/sqlalchemy/dialects/mysql/mysqldb.py", line 105, in do_executemany
    rowcount = cursor.executemany(statement, parameters)
```